spawning.kv

```diff
@@ -7,13 +7,14 @@
     Shot:
         id: turret_shot
         source: 'atlas://turret/shot'
-        size: 32, 32
-        center: root.width / 2, 0
+        allow_stretch: True
```

Comment:
The changes here are to increase the size of the sprites on screen so that they show up more clearly.